demos/MLSys2020/demo.ipynb

```diff
@@ -0,0 +1,116 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import drawSvg as draw\n",
+    "\n",
+    "import plaidml\n",
+    "import plaidml.exec\n",
+    "from plaidml.edsl import *\n",
+    "\n",
+    "\n",
+    "def edsl_program(X, Y):\n",
+    "    I, J, K = TensorDims(3)\n",
+    "    i, j, k = TensorIndexes(3)\n",
+    "    X.bind_dims(I, K)\n",
+    "    Y.bind_dims(K, J)\n",
+    "    R = TensorOutput(I, J)\n",
+    "    R[i, j] += X[i, k] * Y[k, j]\n",
+    "    return R"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#  use placeholder, make them ints\n",
+    "A = Tensor(LogicalShape(plaidml.DType.INT32, [3, 3]))\n",
```

Comment:
@flaub perhaps I'm missing something extremely basic here but it seems like the `Placeholder` API only exists for the C++ EDSL? I didn't see it when I did a `git grep` and I also couldn't find any examples of Python EDSL code using `Placeholder`.

Reply:
The C++ and python APIs should have parity, if it doesn't, could you even them out so that they do?

Reply:
Added this to our checklist. I'll address in a separate PR.